internal/api/v1/services.go

```diff
@@ -107,6 +110,7 @@ func (sc ServicesController) Index(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	appsOf, err := servicesToApps(cluster, gitea, org)
+
```

Comment:
[nit] you probably wanted to add the empty line above `appsOf` not below.

Reply:
Agreed.

Reply:
Commit 17f989c20b754a2dac2b7be2a629e7b9c699079d